octavia_f5/restclient/as3types.py

```diff
@@ -11,3 +11,22 @@
 # WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
 # License for the specific language governing permissions and limitations
 # under the License.
+
+import re
+import unicodedata
+
+
+def f5remark(remark):
+    # Remove control characters.
+    nstr = "".join(ch for ch in remark
+                   if unicodedata.category(ch)[0] != "C")
+
+    # Remove double-quote (“), and backslash (\), limit to 64 characters.
+    return re.sub('["]', '', nstr)[:63]
```

Comment:
Character class must contain backslash: `'["\\\\]'`
(has to be double-escaped for regex-engine and python string evaluation)

Also: Allow 64 characters (`[:64]` instead of `[:63]`)